lib/ae_mdw/db/model.ex

```diff
@@ -663,12 +663,24 @@ defmodule AeMdw.Db.Model do
             index: entity_index()
           )
 
+  # entity_args:
+  #     index: {name, create txi, args}
+  #     txi: initial txi
+  @entity_args_defaults [index: {"", -1, []}, txi: -1]
+  defrecord :entity_args, @entity_args_defaults
+
+  @type entity_args_index() :: {String.t(), txi(), list()}
+  @type entity_args() ::
+          record(:entity_args,
+            index: entity_args_index()
+          )
```

Comment:
It's missing `txi` field here. Also, should the list of arguments be part of the index? Can we move that to another record field?

Reply:
Thanks! Regarding the list on the index, it is needed to be able to fetch the auction for a specific NFT but the list is comprised only by the collection contract id and token id. As we have other records with the index having two pubkeys it won't be a problem.